resources/deb/etc/init.d/openhab2

```diff
@@ -71,15 +71,36 @@ fi
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
 
+# Check to see if openHAB is currently running
+EXISTINGPID=`pgrep java -G openhab | tail -1`
```

Comment:
Good idea. I'm unsure if the filter is precise enough though. Yes the chances are pretty low that another java process is executed as openhab but still... What do you think?